bin_sort/BinSort.java

```diff
@@ -0,0 +1,78 @@
+import java.util.*;
+
+class Node {
+	double data;
+	Node next;

```

Comment:
@Hsankesara This issue by Codacy means you have to explicitly mention `private` before Node